src/components/application_manager/test/commands/hmi/simple_requests_to_hmi_test.cc

```diff
@@ -213,17 +225,7 @@ TYPED_TEST(RequestToHMICommandsTest, Run_SendMessageToHMI_SUCCESS) {
 
   SharedPtr<CommandType> command = this->template CreateCommand<CommandType>();
 
-  EXPECT_CALL(this->mock_app_manager_, SendMessageToHMI(NotNull()));
-
-  command->Run();
-}
-
-TYPED_TEST(RequestToHMICommandsTest2, Run_SendMessageToHMI_SUCCESS) {
```

Comment:
@LevchenkoS I fixed that right after PR. I used a new mergetool and lines were deleted by mistake. It is already fixed as you will see in next commit.